manticore/core/smtlib/visitors.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, **kw):
         BitVecSub: operator.__sub__,
         BitVecMul: operator.__mul__,
         BitVecDiv: operator.__truediv__,
+        BitVecUnsignedDiv: operator.__floordiv__,
```

Comment:
Can we move this to another PR and [possibly] add a unit test for that?

Reply:
It is needed for this PR in that, in some of my tests, the symbolic pc features an unsigned division.

I will move this into a separate commit and add a unit test as you suggested.

Reply:
#1526 has a different implementation of this: 
https://github.com/trailofbits/manticore/pull/1526/files#diff-925668d298ca105732187a108baacbc7R294

@feliam Any thoughts?

Reply:
The implementation in #1526 looks Ethereum specific.  But `manticore/core/smtlib/visitors.py` is not Ethereum specific, is it?